templates/catalog/category.html

```diff
@@ -62,8 +54,23 @@
                      Reverse the url at the views or at the templates.
                   {% endcomment %}
                   <a class="series-filter-link category-filter-link"
-                     href="/gbi/series/{{ series_one.slug }}/category/{{ category.id }}">
-                    {{ series_one.name }}
+                     href="/gbi/series/{{ series_one.page.slug }}/category/{{ category.id }}/">
```

Comment:
We should avoid hardcoded urls, let's reverse them instead

Reply:
@artemiy312 , we have subtask about it few lines ahead. It's tracker version https://github.com/fidals/stroyprombeton/issues/618